### DISCUSSION TITLE
Question on creating a codespace

### DISCUSSION THREAD

**Discussion by SciComp8** [community/community#64896] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Greetings!

Thanks for writing this nice quick start guide for github codespace usage (https://docs.github.com/en/codespaces/getting-started/quickstart). When I click the `Open in a codespace` for [github/haikus-for-codespaces](https://github.com/github/haikus-for-codespaces), my safari browser only shows the Setting up your codespace and a black rectangle below it. What's going on, and do you know how to address this issue? 

I would really appreciate your insight and help. Thank you so much.

**Comment by ggdio** [comment, marked as answer]
Hi there !

I'm sorry to inform you but Safari is no compatible with Visual Studio Web due to lack of some native functionalities that Safari doesnt provide, and as "code space" is basically the visual studio web, then it wont open properly.

I recommend that you use Google Chrome instead.

**Reply by SciComp8** [reply]
Yes, you are right. Thank you so so much for your very helpful recommendation.

**Comment by nengkya** [comment]
I need galaxy brain.
